ntoskrnl/ex/event.c

```diff
@@ -107,6 +107,13 @@ NtCreateEvent(OUT PHANDLE EventHandle,
     DPRINT("NtCreateEvent(0x%p, 0x%x, 0x%p)\n",
             EventHandle, DesiredAccess, ObjectAttributes);
 
+    /* Validate the event type */
+    if ((EventType != NotificationEvent) &&
+        (EventType != SynchronizationEvent))
+    {
+        return STATUS_INVALID_PARAMETER;
```

Comment:
suggestion:
```suggestion
        return STATUS_INVALID_PARAMETER_4;
```
like what's done in NtCreateTimer:
https://github.com/reactos/reactos/blob/fff6d960003fe79b17d6e52875de60d3964e446b/ntoskrnl/ex/timer.c#L383-L389